### DISCUSSION TITLE
How to load few components on Editor load

### DISCUSSION THREAD

**Discussion by RajathShanmugaiah** [react-page/react-page#1039] [Q&A] [answered]
In Editor I have passed all the custom cell plugins and is also bieng rendered smoothly on the component drawer. 

My question Now is ,
For eg. I have lot of components Hero, footer, images text section etc etc. 

So basically what I want to achieve is when the page is rendered I want the the Hero and Footer loaded initially. So in between header and footer I can add N number of components.

**Comment by macrozone** [comment, marked as answer]
@RajathShanmugaiah I don't understand your question. Do you mean you need a default value when the editor has no data yet?

**Reply by RajathShanmugaiah** [reply]
@macrozone 
So Now basically when i do **npm run dev**  my index is loaded which has the editor  with my custom cell plugins asking me to add the components from the components drawer.

What I want to achieve and my query is can the editor be loaded with default components on the initial load?

**Reply by macrozone** [reply]
you will usually load and save the data of the editor somewhere. If your page, your app, your blogpost or whatever you want to do is empty, you could just pass some default value.

The various examples (https://github.com/react-page/react-page/tree/master/examples/pages/examples) use initial values.
We introduced a helper some time ago, that makes it a bit simpler to create values:

```
import { createValue } from '@react-page/editor';
```

you can see it here: https://github.com/react-page/react-page/blob/master/examples/pages/examples/stretchCells.tsx#L38

`createValue` is slightly less strict than typing the whole `Value` by hand, e.g. you don't need to invent ids for node (these are generated), etc. I highly recommend to use typescript, as this makes it easier to create the right argument for `createValue` as it can still be a bit a complex data structure.

**Reply by roker15** [reply]
How to increase font size and change types? Is it even possible ? i see no options

**Reply by macrozone** [reply]
please see https://react-page.github.io/docs as this discussion here is unrelated to your question
